doc/docker_quick_start.rst

```diff
@@ -0,0 +1,63 @@
+Docker Quick Start
+==========================
+
+How to add a new asset to Ralph and discover information about the added device in Docker Container
```

Comment:
Enter more introduction, similar to https://docs.djangoproject.com/en/1.7/intro/tutorial01/